docs/source/contributing.rst

```diff
@@ -33,17 +34,101 @@ If you're unsure about any of these points, please open the Pull Request anyhow
 Pull Request Process
 --------------------
 
-After you open the Pull Request, there will probably be some discussion in the comments' field of the request itself.
+After you open the Pull Request, there will probably be some discussion in the comments field of the request itself.
 
 Once the Pull Request is ready to merge, it will first be merged into our internal git system for in-house automated testing.
 
-If this process passes, it will be merged onto the public github repository.
+If this process passes, it will be merged into the public GitHub repository.
 
-Legal Part
-----------
+Example Contribution Guideline
+------------------------------
+
+Checklist
+*********
+
+* Check if your example proposal has no similarities to the project
```

Comment:
I will add here this to make it more clear:

* Check if your example proposal has no similarities to the project **(already existing examples)**